contracts/extensions/BasicIssuanceExtension.sol

```diff
@@ -0,0 +1,229 @@
+
+import { BaseGlobalExtension } from "../lib/BaseGlobalExtension.sol";
+import { IDelegatedManager } from "../interfaces/IDelegatedManager.sol";
+
+import { IIssuanceModule } from "../interfaces/IIssuanceModule.sol";
+
+/**
+ * @title BasicIssuanceExtension
+ * @author Set Protocol
+ *
+ * Smart contract global extension which provides DelegatedManager owner and 
+ * methodologist the ability to accrue and split issuance and redemption fees at an mutable percentage.
+ */
+contract BasicIssuanceExtension is BaseGlobalExtension {
+    using Address for address;
+    using PreciseUnitMath for uint256;
+    using SafeMath for uint256;
+
+    /* ============ Events ============ */
+
+    event ExtensionInitialized(
+        address _setToken,
+        address _delegatedManager
+    );
+
+    event FeesDistributed(
+        address _setToken,
+        address indexed _ownerFeeRecipient,
+        address indexed _methodologist,
+        uint256 _ownerTake,
+        uint256 _methodologistTake
+    );
+
+    /* ============ State Variables ============ */
+
+    // Instance of BasicIssuanceModule
+    IIssuanceModule public immutable issuanceModule;
+
+    // Mapping from Set Token to DelegatedManager 
+    mapping(ISetToken => IDelegatedManager) public setManagers;
+
+    /* ============ Constructor ============ */
+
+    constructor(
+        IIssuanceModule _issuanceModule
+    )
+        public
+    {
+        issuanceModule = _issuanceModule;
+    }
+
+    /* ============ External Functions ============ */
+
+    /**
+     * ANYONE CALLABLE: Distributes fees accrued to the DelegatedManager. Calculates fees for
+     * owner and methodologist, and sends to owner fee recipient and methodologist respectively.
+     */
+    function distributeFees(ISetToken _setToken) public {
+        uint256 totalFees = _setToken.balanceOf(address(_manager(_setToken)));
+
+        address methodologist = _manager(_setToken).methodologist();
+        address ownerFeeRecipient = _manager(_setToken).ownerFeeRecipient();
+
+        uint256 ownerTake = totalFees.preciseMul(_manager(_setToken).ownerFeeSplit());
+        uint256 methodologistTake = totalFees.sub(ownerTake);
+
+        if (ownerTake > 0) {
+            _setToken.transfer(ownerFeeRecipient, ownerTake);
+        }
+
+        if (methodologistTake > 0) {
+            _setToken.transfer(methodologist, methodologistTake);
+        }
+
+        emit FeesDistributed(address(_setToken), ownerFeeRecipient, methodologist, ownerTake, methodologistTake);
+    }
+
+    /**
+     * ONLY OWNER: Initializes BasicIssuanceExtension to the DelegatedManager.
+     *
+     * @param _delegatedManager     Instance of the DelegatedManager to initialize
+     */
+    function initializeExtension(IDelegatedManager _delegatedManager) external {
+        require(msg.sender == _delegatedManager.owner(), "Must be owner");
+        require(_delegatedManager.isPendingExtension(address(this)), "Extension must be pending");
```

Comment:
do we want to require that the global extension's corresponding module is initialized?
- con: an owner can successfully call initializeExtension when the module is not initialized. They're extension will be initialized but no features on the modules will work
- pro: maybe some migration flow works if this require is not present?